File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int x =0;
        int n = nums.length;
        for(int i =1;i<n;i++){
            if(nums[i]>nums[x]){
                x++;
                nums[x] = nums[i];
            }
        }
        return x+1;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let x =0;
    let n = nums.length;
    
    for(let i=1;i<n;i++){
        if(nums[i]>nums[x]){
            x++;
            nums[x] = nums[i];
        }
    }
    return x+1;
};